packages/client/src/pages/IntroPage.test.tsx

```diff
@@ -0,0 +1,131 @@
+import "@testing-library/jest-dom/extend-expect";
+
+import React from "react";
+
+import nl from "../i18n/nl";
+import { Topic } from "../types";
+import { NEXT_BUTTON } from "../utils/test-ids";
+import { render, screen, waitFor } from "../utils/test-utils";
+import IntroPage from "./IntroPage";
+
+const mockIntro: Topic = {
+  hasIMTR: true,
+  intro: "DakkapelIntro",
+  name: "Dakkapel plaatsen",
+  slug: "dakkapel-plaatsen",
+  text: {
+    heading: "Vergunningcheck dakkapel plaatsen",
+    locationIntro: "Voer het adres in waar u de dakkapel wilt gaan plaatsen",
+  },
+};
+
+jest.mock("./../hooks/useTopic", () => {
+  return jest.fn(() => mockIntro);
+});
```

Comment:
This doesn't do anything, since the `dakkapel-plaatsen` topic is loaded by default.